index.html

```diff
@@ -1638,8 +1638,7 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
       attribute EventHandler onreading;
       attribute EventHandler onerror;
 
-      void start();
-      void stop();
+      void scan(optional NFCScanOptions options={});
```

Comment:
We probably want to move the ctor options into this

Reply:
Done.